bin/download.tk1-exec-uboot

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+if [ -n "${board_usb_dev}" ]; then
+    link=`readlink "${board_usb_dev}"`
+    instance=`echo "${link}" | sed -e "s@bus/usb@@"`
+    path="--usb-port-path ${instance}"
+fi
```

Comment:
This if block doesn't seem useful, since board_usb_dev is not set in the TK1 config added by this patch.

Reply:
Yes...but if we get it working...

Reply:
If we get what working? You mean udev? In that case, this block still doesn't seem necessary; I have TK1 working on my system without this block, so I'm not sure what it's meant to do or why it's needed.

Reply:
OK I just copied it from one of the other scripts.